### PR TITLE
Update sensitive-information-type-entity-definitions.md

### PR DESCRIPTION
**Pull request by msbemba**
adding a "what is new?" section to keep track of newly added or changed items in the page so that it keeps it easier to track as there is a lot of information that gets changed or added frequently

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2486

File: microsoft-365/compliance/sensitive-information-type-entity-definitions.md [hunk 1]
```diff
@@ -12702,3 +12702,9 @@ A DLP policy is 75% confident that it's detected this type of sensitive informat
 - Passeport# 
 - PasseportNon 
 - Passeportn ° 
+
+
+# What is New?
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-entity-definitions.md]:
```suggestion
# What is new?
```

File: microsoft-365/compliance/sensitive-information-type-entity-definitions.md [hunk 2]
```diff
@@ -12702,3 +12702,9 @@ A DLP policy is 75% confident that it's detected this type of sensitive informat
...
+
+This section will have the newly added.Changed items to make it easier to keep track of changes to the Sensitive Info Types
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/sensitive-information-type-entity-definitions.md]:
```suggestion
This section will contain newly added changed items to make it easier to keep track of changes to the sensitive info types.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Comment by chrfox** [comment]
This topic is highly structured, no changes will be made without involving engineering and content development

**Pull request closed without merging**